test/adapter/json_api/links_test.rb

```diff
@@ -40,7 +40,7 @@ def test_toplevel_links
                   stuff: 'value'
                 }
               }
-            }).serializable_hash
+            }).serializable_hash(@options)
```

Comment:
there's a bug in this right now https://github.com/rails-api/active_model_serializers/pull/1572